development/tools/jobs/kyma/kyma_integration_test.go

```diff
@@ -69,6 +69,23 @@ func TestKymaIntegrationJobsPresubmit(t *testing.T) {
 				"installation/test/test/README.MD",
 			},
 		},
+		"Should contain the kyma-integration k3d with telemetry job": {
+			givenJobName: "pre-main-kyma-integration-k3d-telemetry",
+
+			expPresets: []preset.Preset{
+				preset.GCProjectEnv, preset.KymaGuardBotGithubToken, preset.BuildPr, "preset-sa-vm-kyma-integration", "preset-kyma-integration-telemetry-enabled",
+			},
+
+			expRunIfChangedRegex: "^components/telemetry-operator/|^resources/telemetry/",
+			expRunIfChangedPaths: []string{
+				"components/telemetry-operator/values.yaml",
```

Comment:
I guess this should be "resources/telemetry/charts/operator/values.yaml"? Maybe also add the values file of the fluent-bit subchart.